src/nimdowpkg/windowmanager.nim

```diff
@@ -951,6 +959,22 @@ proc onConfigureRequest(this: WindowManager, e: XConfigureRequestEvent) =
 
   discard XSync(this.display, false)
 
+proc onConfigureNotify(this: WindowManager, e: XConfigureEvent) =
+  if e.window == this.rootWindow:
+    log "rootWindow onConfigureNotify"
```

Comment:
Could we remove the log statement?

Reply:
I'll fix that.

> Thanks for making the PR! What behaviors have you tested as far as randr events go?

Tested resizing with both xephyr and by moving nimdow between my laptop and an external display while running on a laptop. It won't add/remove displays but it handles the resolution changing.